getrepodata.py

```diff
@@ -0,0 +1,88 @@
+import asyncio
+from collections import defaultdict
+
+import tqdm
+import pydriller
+import httpx
+import itertools
+
+headers = {'Authorization': 'token '}
```

Comment:
let's follow pep8 - constants should be `HEADERS` in uppercase
```suggestion
HEADER= {'Authorization': 'token '}
```